app/assets/javascripts/subject.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
as I understand this file was generated by `rails g scaffold`, but there is no point of having it in repo since it's empty

Reply:
added to .gitignore, removed from repo